runtime/oti/j9vm.hdf

```diff
@@ -1299,4 +1299,18 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		<data type="struct J9VMThread*" name="vmThread" description="current thread" />
 		<data type="U_32" name="state" description="the VM runtime state" />
 	</event>
+	
+	<event>
+		<name>J9HOOK_VM_ZIP_OPEN_STATE</name>
```

Comment:
I don't think we should use a hook, it isn't necessary. The hookZipOpenState() function can be added to the J9SharedClassConfig and called that way.

Reply:
Agree and already removed the hook related code/setting.